docs/topics/secured-booster-minishift.adoc

```diff
@@ -1,13 +1,13 @@
 = Configuring Your {OpenShiftLocal}
-Before you can run the Secured Booster, you must update your {OpenShiftLocal} from the default configuration. 
+Before you can run the Secured booster, you must update your {OpenShiftLocal} from the default configuration. 
 You only need to do this once for any of the secured mission boosters.
 They all share the same {RHSSO}/{OpenShiftLocal} setup.
 
 Before you can use your {OpenShiftLocal}, you need to have it installed, configured, and running. You can find details on installing a {OpenShiftLocal} for your platform in link:{link-launcher-openshift-local-install-guide}#installing-a-openshiftlocal[{minishift-installation-guide-name}].
 
 .Updating your {OpenShiftLocal} from the default Configuration
 
-{RHSSO} will not start up unless you switch from the default configuration. The SSO Booster currently only works with the CentOS base image. If you have already run the `minishift` command with a different memory setting and iso-url value,
+{RHSSO} will not start up unless you switch from the default configuration. The SSO booster currently only works with the CentOS base image. If you have already run the `minishift` command with a different memory setting and iso-url value,
```

Comment:
Do we still have this? It's no longer true that you have to use the CentOS image. Maybe we didn't remove all the mentions of this.

Reply:
@Ladicek. You're right, my understanding is that this restriction no longer applies. We've also removed the related callout from the Secure HTTP API mission docs.

Reply:
I know we removed the callout, maybe we should remove it from here too :-) Can be in a different PR, I don't mind.

Reply:
Agreed:) should definitely remove it from here, too:)

Reply:
Sorry I missed this. Will fix in separate PR.

Reply:
@Ladicek @inoxx03 as a followup, this topic is no longer used in general, so i'll just remove it.

Reply:
PR: https://github.com/fabric8-launcher/launcher-documentation/pull/1056